studio/schemas/documents/backgroundImgwithCard.js

```diff
@@ -18,5 +18,73 @@ export default {
             title: 'Background Image',
             type: 'image',
         },
+        {
+            name: 'backgroundColor',
+            title: 'Background Color',
+            type: 'string',
+            description:
+                'Selection will give the card a white, dark blue or transparent background color. The default color is white',
+            options: {
+                list: [
+                    { title: 'Light', value: 'light' },
+                    { title: 'Dark', value: 'dark' },
+                    { title: 'Transparent', value: 'transparent' },
+                ],
+            },
+        },
+        {
+            name: 'cardOrientation',
+            title: 'Card Orientation',
+            type: 'string',
+            description:
+                'Selection will align the card to the right or left of the container. Default is left oriented',
+            options: {
+                list: [
+                    { title: 'Right', value: 'right' },
+                    { title: 'Left', value: 'left' },
+                ],
+            },
+        },
+        {
+            name: 'button',
+            title: 'Button',
+            type: 'string',
+            description: 'Selection will give card a button',
+            options: {
+                list: [{ title: 'Button', value: 'button' }],
+            },
+        },
```

Comment:
I don't think you need this field

Reply:
I think we do, hasButton tells the component it needs a button. {hasButton &&....)

Reply:
You could derive a `hasButton` variable based on some other state that is returned from the query. You don't need to have a `button` field in sanity as well.